src/app/app-components/courses/add-course/add-course.component.ts

```diff
@@ -42,18 +51,34 @@ export class AddCourseComponent implements OnInit {
     const timestamp = today.getTime().toString();
     this.courseInfo = {
       id: timestamp,
-      title: this.title,
-      creationDate: new Date(),
-      duration: this.duration,
+      name: this.name,
+      date: new Date(),
+      length: this.length,
       description: this.description,
-      topRated: false,
+      isTopRated: false,
     };
     this.save.emit(this.courseInfo);
 
     if (this.savingMode === SavingModes.CREATE) {
-      this.coursesService.create(this.courseInfo);
+      // ToDo: Implement create course in BE
+      this.httpClient.post(`${this.BASE_URL}/courses`, {
+        id: this.courseInfo.id,
+        name: this.courseInfo.name,
+        date: this.courseInfo.date,
+        length: this.courseInfo.length,
+        description: this.courseInfo.description,
+        isTopRated: this.courseInfo.isTopRated,
+      }).subscribe();
```

Comment:
We can use the many `subscribe` callbacks to handle the result of the HTTP call. For example:

```typescript
...
.subscribe(
    (response) => { /* handle success */},
    (error) => { /* handle the error */ }
);
```

Reply:
Done.